src/assets/scripts/groepen/ProfielLink.jsx

```diff
@@ -0,0 +1,93 @@
+const React = require("react");
+const $ = require("jquery");
+const _ = require("underscore");
+const api = require("api");
+const PropTypes = require('react-router').PropTypes;
+const Civikaartje = require("./Civikaartje");
+
+class ProfielLink extends React.Component {
+
+	static get propTypes() {
+    return {
```

Comment:
Uh, indenting should be fixed! Not just here, everywhere in this file (2 spaces)